integration/tests/cook/test_multi_user.py

```diff
@@ -338,62 +338,6 @@ def is_rate_limit_triggered(_):
             finally:
                 util.kill_jobs(self.cook_url, job_uuids)
 
-    # Note that subsequent runs of this test under the same user can fail if sufficient time has not
-    # passed; the subsequent run will have used up the rate limit quota and it will need time to recharge.
-    def test_global_rate_limit_launching_jobs(self):
```

Comment:
Is this test worth migrating? It's coupled to travis and mesos, because only there do we have an easy ability to create an extra cook cluster with a very low global quota, which this test triggers. I decided to might as well delete, but I can migrate instead.

Reply:
If we move the rate-limiting configuration into the compute cluster config template (I proposed this in a separate comment in this PR), could we have a test where we change the template on a cluster to a template that has much lower rate limits, and then assert we got rate-limited? I'm not sure if it's possible, @nsinkov, any thoughts?

Reply:
Moving the config knob elsewhere doesn't make it dynamic. We want a (low rate limit configuration) for the single test and a normal rate limit for everything else. How do we get that without bringing up two versions of Cook, or an extra compute cluster & corresponding pool?

Reply:
if the test is using the mesos cluster only, then it won't use templates.

Am I missing something?

Reply:
My suggestion was for an integration test of the rate limiting functionality, not necessarily this exact test. 

@nsinkov, could we have two templates, one with the normal rate limit, and the other with a much lower rate limit, and make the test change the template of the k8s clusters to assert that the rate limits get applied? And then change the template back to the normal rate limit one?

Reply:
Are templates dynamically changable?

Reply:
@nsinkov?

Reply:
I think the plan is yes, we will try to sue templates and use the dynamic configuration when it is working. We can create and disable a compute cluster --- enable a compute cluster with low limits, disabling the rest, run the tests, then disable it and reenable the original ones. We can avoid having a separate k8s cluster by reusing it across the two different k8s clusters. Plan is to defer that integration test until after the dynamic compute cluster tests are in and then evaluate how fragile the test will be and how muich work before we implement it.